tests/specifications/null-type.yaml

```diff
@@ -62,3 +80,18 @@ components:
     BlankEnum:
       enum:
         - ''
+    MultipartFormDataNullable:
```

Comment:
It's okay to use an existing schema 👍 

https://github.com/orval-labs/orval/blob/master/tests/specifications/null-type.yaml#L41-L48

Reply:
Thanks, I fixed it.
I also found a bug when `oneOf, anyOf, allOf` subschemas are attached and fixed it.